lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -594,8 +594,8 @@ class EngineParagraph implements ui.Paragraph {
     }
 
     final int start =
-        WordBreaker.prevBreakIndex(_plainText, textPosition.offset);
-    final int end = WordBreaker.nextBreakIndex(_plainText, textPosition.offset);
+        WordBreaker.prevBreakIndex(_plainText!, textPosition.offset + 1);
+    final int end = WordBreaker.nextBreakIndex(_plainText!, textPosition.offset);
```

Comment:
Is enforcing non-null the desirable semantic? Should we default to empty string, or make `_plaintText` non-nullable?

Reply:
A few lines above, we do check for `_plainText == null`. So at this point we are sure `_plainText` isn't null. Would it be better if I assign it to a local variable so that type promotion works?